src/move/econia/sources/market.move

```diff
@@ -5150,8 +5330,27 @@ module econia::market {
             restriction, self_match_behavior);
         // Check if order is tail of corresponding price level.
         assert!(is_local_tail_test(market_id, side, market_order_id), 0);
+        // Assert event streams.
+        assert!(user::get_cancel_order_events_test(
```

Comment:
Here too with the confusing cancel asserts.

Reply:
Same rationale as for https://github.com/econia-labs/econia/pull/366#discussion_r1277962396